assets/app/view/create_game.rb

```diff
@@ -3,6 +3,8 @@
 require 'game_manager'
 require 'view/form'
 
+VISIBLE_GAMES = (ENV['RACK_ENV'] == 'production' ? Engine::VISIBLE_GAMES : Engine::GAME_METAS).sort
```

Comment:
Looks like Opal can't access `ENV['RACK_ENV']` here--I started up a local "production" server and `ENV` is `{}` so `VISIBLE_GAMES` here gets set to all of the games.

`api.rb` sets a `PRODUCTION` constant ([here](https://github.com/tobymao/18xx/blob/90baf49a2a3619aa62dbff1077eb2ac3114312f9/api.rb#L3)), that could be added to the needs [here](https://github.com/tobymao/18xx/blob/90baf49a2a3619aa62dbff1077eb2ac3114312f9/api.rb#L139-L146) and [here](https://github.com/tobymao/18xx/blob/90baf49a2a3619aa62dbff1077eb2ac3114312f9/assets/app/app.rb#L29-L30)

Reply:
https://github.com/yzemaze/18xx/pull/3/files